modules/network-observability-without-loki.adoc

```diff
@@ -0,0 +1,19 @@
+// module included in the following assemblies:
+// networking/network_observability/installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-without-loki_{context}"]
+= Network Observability without Loki
+You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:
+
+.Comparison of feature availability with and without Loki
+[options="header"]
+|============================================================================================================================
+|                                     | *With Loki* | *Without Loki* 
+| *Exporters*                         | X           | X      
+| *Flow-based dashboards*             | X           | X       
+| *Traffic Flow Overview, Table and Topology views* | X           | N/A     
+| *Quick Filters*                     | X           | N/A    
+| *{product-title} console Network Traffic tab integration* | X           | N/A
+|============================================================================================================================
```

Comment:
Thinking out loud, would it be good to add a link on how to set up the exporters here so that it would be easier for a customer to set it up?

Reply:
Good idea. I can add it to the assembly file as an additional resource situated directly under this.

Reply:
This link is already an additional resource at the end of the page, but I'm going to move it up to this topic instead.